index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
```

Comment:
You have horizontal scrolling, it is not good

Reply:
Now I see it. Can you give me some pieces of advice how to avoid it?

Reply:
In footer you use position relative and left, so as a result width of the element is not change. If you use flex or inline-block, you don't this problem